sdcm/results_base.html

```diff
@@ -140,7 +140,40 @@
 {% endblock %}
 
 {% block links %}
-
+    <h3>Links:</h3>
+    <ul>
+        {% block extra_link %}
+        {% endblock %}
+        {% if grafana_screenshots %}
+            {% if grafana_screenshots[0] %}
+                <li><a href={{ grafana_screenshots[0] }}>Download "Per server metrics nemesis" Grafana Screenshot</a></li>
+            {% endif %}
+            {% if grafana_screenshots[1] %}
+                <li><a href={{ grafana_screenshots[1] }}>Download "Overview metrics" Grafana Screenshot</a></li>
+            {% endif %}
+            {% if grafana_screenshots[2] %}
```

Comment:
please check if we didn't fail with Index out of range error

Reply:
I've check, and this would return False, if IndexError is raised